### ISSUE TITLE
Bugs in generating wsj0-2mix dataset

### ISSUE THREAD

**Issue by hangtingchen** [mpariente/asteroid#164]
## 🐛 Bug

Should use mv1 instead of mv2 to get wav files

### To Reproduce

https://github.com/mpariente/asteroid/blob/0bdec2644f2d770d037ce804b7f70cb98bd5c9fa/egs/wsj0-mix/DeepClustering/local/convert_sphere2wav.sh#L31

The line uses both mv2 and mv1 to get wav files. But the mv1 will be covered by mv2, resulting in the generated wav files being from mv2. The mv1 is noise-free while the mv2 is noisy. The wsj0-mix dataset is expected to use mv1.

The correct code is 
``
wav=`echo "$line" | sed "s:wv1:wav:g" | awk -v dir=$wav_dir -F'/' '{printf("%s/%s/%s/%s", dir, $(NF-2), $(NF-1), $NF)}'`
``

### Expected behavior

We tested the datasets generated by mv1 and mv2. It is observed that the former can reproduce the results, the latter is worse around 1-2 dB in SI-SNR.

Our results with mv1, the final validation loss was about 2950.
![微信截图_20200709163058](https://user-images.githubusercontent.com/30209378/87016695-a065bf80-c201-11ea-955d-3c603cdf6ac0.png)

I am sorry that our results with mv2 were deleted, its final validation loss was about 3500.

### Environment

 - Asteroid-master
 - PyTorch 1.4.0
 - PyTorchLightning 7.6.1

**Comment by mpariente** [comment]
I think this is right and has been reported elsewhere as well. 
I think I only had wv1 when I generated wsj0 in the first place so I didn't notice the problem. 
I checked now and you're right. 

Would you like to submit a PR for that please? 
Thanks again !

**Comment by mpariente** [comment]
Closed by #166
